vendor/bat-native-ledger/src/bat/ledger/internal/request/request_publisher.cc

```diff
@@ -3,19 +3,34 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+#include "base/strings/string_util.h"
 #include "base/strings/stringprintf.h"
-#include "bat/ledger/internal/static_values.h"
 #include "bat/ledger/internal/request/request_publisher.h"
 #include "bat/ledger/internal/request/request_util.h"
 
+namespace {
+
+constexpr char kPrefix[] = "";
```

Comment:
I would say that this can be removed, based on the fact that @yachtcaptain23 said that we will not do versioning, but protobuff will handle it for us

Reply:
done